src/isSymbol.js

```diff
@@ -0,0 +1,32 @@
+import { type } from 'ramda';
+import curry1 from 'ramda/src/internal/_curry1';
+
+/**
+ * Checks if input value is a Symbol.
+ *
+ * @func isSymbol
+ * @memberOf RA
+ * @since {@link https://char0n.github.io/ramda-adjunct/2.22.0|v2.22.0}
```

Comment:
I had to guess the version based on current version.

Reply:
Version is correct, sorry for this. This will be automated in future.